modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -267,77 +242,6 @@ To see additional optional parameters and explanations of how to use them, run t
 ====
 endif::azure-workload-id[]
 
-ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
```

Comment:
Should this command remain in the updates module? https://75756--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/preparing_for_updates/preparing-manual-creds-update#cco-ccoctl-upgrading_preparing-manual-creds-update (step 1)

Reply:
There are no changes to Updating book, confirmed by PM. Customer can follow the same procedures.